packages/core-modules/contracts/interfaces/IElectionModule.sol

```diff
@@ -16,65 +18,79 @@ interface IElectionModule {
 
+    /// @notice Upgrades the implementation of the existing council NFT token
     function upgradeCouncilToken(address newCouncilTokenImplementation) external;
 
+    /// @notice Adjusts the current epoch schedule requiring that the current period remains Administration, and that changes are small (see setMaxDateAdjustmentTolerance)
     function tweakEpochSchedule(
         uint64 newNominationPeriodStartDate,
         uint64 newVotingPeriodStartDate,
         uint64 newEpochEndDate
     ) external;
 
+    /// @notice Adjusts the current epoch schedule requiring that the current period remains Administration
     function modifyEpochSchedule(
         uint64 newNominationPeriodStartDate,
         uint64 newVotingPeriodStartDate,
         uint64 newEpochEndDate
     ) external;
 
+    /// @notice Determines minimum values for epoch schedule adjustments
     function setMinEpochDurations(
         uint64 newMinNominationPeriodDuration,
         uint64 newMinVotingPeriodDuration,
         uint64 newMinEpochDuration
     ) external;
 
+    /// @notice Determines adjustment size for tweakEpochSchedule
     function setMaxDateAdjustmentTolerance(uint64 newMaxDateAdjustmentTolerance) external;
 
+    /// @notice Determines batch size when evaluate() is called with numBallots = 0
     function setDefaultBallotEvaluationBatchSize(uint newDefaultBallotEvaluationBatchSize) external;
 
+    /// @notice Determines the number of council members in the next epoch
     function setNextEpochSeatCount(uint8 newSeatCount) external;
 
+    /// @notice Determines the minimum number of council members before triggering an emergency election
     function setMinimumActiveMembers(uint8 newMinimumActiveMembers) external;
 
+    /// @notice Allows the owner to remove one or more council members, triggering an election if a threshold is met
     function dismissMembers(address[] calldata members) external;
 
     // ---------------------------------------
     // User write functions
     // ---------------------------------------
 
+    /// @notice Allows anyone to self-nominate during the Nomination period
     function nominate() external;
 
+    /// @notice Self-withdrawal of nominations during the Nomination period
     function withdrawNomination() external;
 
+    /// @notice Allows anyone with vote power to vote on nominated candidates during the Voting period
     function cast(address[] calldata candidates) external;
 
+    /// @notice Allows votes to be withdraw
     function withdrawVote() external;
 
+    /// @notice Processes ballots in batches during the Evaluation period (after epochEndDate)
     function evaluate(uint numBallots) external;
 
+    /// @notice Shuffles NFTs and resolves an election after it has been evaluated
     function resolve() external;
 
     // ---------------------------------------
     // View functions
     // ---------------------------------------
 
-    // Settings
-    // ~~~~~~~~~~~~~~~~~~
-
```

Comment:
Removed these section comments. They were starting to annoy me because some functions don't intuitively fall into these imposed "categories".